source/blog/2018-12-14-the-ember-times-issue-77.md

```diff
@@ -1,18 +1,24 @@
 ---
-title: The Ember Times - Issue No. XX
-author: the crowd
+title: The Ember Times - Issue No. 77
+author: Jessica Jordan, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
 alias : "blog/2018/12/14-the-ember-times-issue-77.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+This week **boolean component arguments** 0️⃣1️⃣ are in for a new **RFC**,
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Boolean Dreams Come True 0️💭⃣1️⃣](https://github.com/emberjs/rfcs/pull/407)
+
+Ever wanted to pass **boolean arguments** to your **components** _just_ like you would pass boolean attributes - like checked or readonly - to a HTML element?
+
+This fresh [RFC (Request for Comments) discusses how this could become reality](https://github.com/hakubo/rfcs/blob/component-boolean-arguments/text/0000-component-boolean-arguments.md)!
```

Comment:
qq any reason why we spell out what RFC means?

Reply:
I add the full description for any acronyms used in each newsletter, because I assume that not everyone reading the newsletter is already familiar with these terms, e.g. because they're new to Ember (or Rust / React which have adopted RFCs as part of their decision processes) and I'd like to make the language as approachable as possible